src/CoreBundle/Resources/views/snippets-cms/TCMSFieldWYSIWYG/cKEditor/editor.html.twig

```diff
@@ -1,4 +1,11 @@
-<div>
+<div
+    data-field-version-history-managed
+    data-field-configuration-id="{{ fieldConfigurationId | e('html_attr') }}"
+    data-field-has-version-history="{{ fieldHasVersionHistory | e('html_attr') }}"
+    data-field-version-history-title="Versionshistorie"
+    data-field-number-of-field-versions="{{ fieldVersionHistoryNumberOfVersions | e('html_attr') }}"
+    data-field-version-history-view-url="{{ fieldVersionHistoryViewUrl | e('html_attr') }}"
```

Comment:
Similarly to the note above, we could think about having this URL shared, but I think it's OK to keep for now.